src/bundle/Resources/views/admin/search/search_form.html.twig

```diff
@@ -0,0 +1,9 @@
+{{ form_start(form) }}
+<div class="input-group">
+    {{ form_widget(form.query, {'attr': {'style': 'width: 200px', 'class': 'form-control'} }) }}
```

Comment:
We have to move inline `style` to CSS. @dew326 can propose you best solution.

Reply:
Inline style is not needed anymore